docs/guides/training-predictions.md

```diff
@@ -0,0 +1,161 @@
+# Training predictions
+
+In the **Skforecast** library, training predictions can be obtained either by using the `backtesting_forecaster()` function or by using the `predict()` method on the regressor of the forecaster object.
```

Comment:
"or by accessing the `predict` method of the regressor stored inside the forecaster object."